libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/IpvSessionService.java

```diff
@@ -101,14 +100,14 @@ public IpvSessionItem generateIpvSession(
 
         ipvSessionItem.setCreationDateTime(Instant.now().toString());
 
-        ipvSessionItem.setUserState(generateStartingState(errorObject));
-
         ipvSessionItem.setVot(Vot.P0);
 
-        String journeyType = this.configService.getSsmParameter(ConfigurationVariable.JOURNEY_TYPE);
-        ipvSessionItem.setJourneyType(IpvJourneyTypes.valueOf(journeyType));
-
-        if (errorObject != null) {
+        if (errorObject == null) {
+            ipvSessionItem.setJourneyType(IpvJourneyTypes.INITIAL_JOURNEY_SELECTION);
+            ipvSessionItem.setUserState(START_STATE);
+        } else {
+            ipvSessionItem.setJourneyType(IpvJourneyTypes.TECHNICAL_ERROR);
+            ipvSessionItem.setUserState(ERROR_STATE);
```

Comment:
I couldn't find `FAILED_CLIENT_JAR` anywhere, so I've routed this to the technical error state. I'm assuming it would have done that anyway, but via a 'unknown state'...